canonical-web-and-design/snapcraft.io
# Validate individual GitHub repos

## Done

Added an endpoint that return the status of a GitHub repo

## Issue / Card

Fixes #1240

## QA

- Pull the branch
- Run the site using the command `./run`
- View the site locally in your web browser at: http://0.0.0.0:8004/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Ask Fran for the .env.local file
- Login as a publisher and check that this endpoint is working:
http://0.0.0.0:8004/<snap_name>/builds/validate-repo?repo=github_owner/repo_name
Like:
http://0.0.0.0:8004/toto-fran/builds/validate-repo?repo=jkfran/toto-fran
